### DISCUSSION TITLE
article/java-21-virtaul-threads

### DISCUSSION THREAD

**Discussion by giscus** [dyc87112/giscus-didispace#4] [Announcements]
# article/java-21-virtaul-threads

启动1000万个虚拟线程需要多少时间？需要多少平台线程？ 之前，在Java新特性专栏中，我们简单介绍了Java 21正式发布的虚拟线程。 昨天，正好看到一个讲解此内容的视频，非常不错，所以DD这里给大家翻译好了，感兴趣的可以看看。可以进一步了解虚拟线程。

https://www.didispace.com/article/java-21-virtaul-threads.html

**Comment by wxa56** [comment]
我这1000万12秒😄
